packages/blocks/drawing-canvas/src/utils.ts

```diff
@@ -0,0 +1,51 @@
+import { TDDocument, TDExport, TldrawApp } from "@tldraw/tldraw";
+
+/**
+ * @todo implement endpoint for handling image exports
+ * This only handles svg exports at the moment
+ * @see https://github.com/tldraw/tldraw/blob/main/apps/www/utils/export.ts
+ * @see https://github.com/tldraw/tldraw/blob/main/apps/www/pages/api/export.ts
+ */
+
+export async function handleExport(info: TDExport) {
```

Comment:
Minor: let's use `=>` fn style for consistency

Reply:
Thanks. Added here => https://github.com/hashintel/hash/pull/515/commits/229dfe48559f79788dac315167cd7ad95af46289